### ISSUE TITLE
[BUG/Help] <title>

### ISSUE THREAD

**Issue by nanisnull** [THUDM/ChatGLM-6B#1400]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

在使用chatGLM-6B-PT进行一个只有6条数据集的微调时，在推理阶段预测值全为0。请问这是因为模型参数设置有问题还是数据集设置有问题呢。

这是results.json
{
    "epoch": 2400.0,
    "predict_bleu-4": NaN,
    "predict_rouge-1": NaN,
    "predict_rouge-2": NaN,
    "predict_rouge-l": NaN,
    "predict_runtime": 1.4923,
    "predict_samples": 6,
    "predict_samples_per_second": 4.021,
    "predict_steps_per_second": 4.021,
    "train_loss": 0.0006010692119598389,
    "train_runtime": 9556.3384,
    "train_samples": 6,
    "train_samples_per_second": 5.023,
    "train_steps_per_second": 0.314
}

这是生成预测文本：
{"labels": "简约而不简单的牛仔外套，白色的衣身十分百搭。衣身多处有做旧破洞设计，打破单调乏味，增加一丝造型看点。衣身后背处有趣味刺绣装饰，丰富层次感，彰显别样时尚。", "predict": ""}
{"labels": "你好，我是菜菜", "predict": ""}
{"labels": "你好，我是菜菜，一个帅气的偶像。", "predict": ""}
{"labels": "菜菜是一个帅气的偶像，擅长唱，跳，篮球和rap。最出名的歌曲是及你太美。", "predict": ""}
{"labels": "菜菜是一个帅气的偶像，擅长唱，跳，篮球和rap。最出名的歌曲是及你太美。", "predict": ""}
{"labels": "菜菜是一个帅气的偶像，擅长唱，跳，篮球和rap。最出名的歌曲是及你太美。", "predict": ""}

这是训练和测试文本：
[{"content": "类型#上衣*材质#牛仔布*颜色#白色*风格#简约*图案#刺绣*衣样式#外套*衣款式#破洞", "summary": "简约而不简单的牛仔外套，白色的衣身十分百搭。衣身多处有做旧破洞设计，打破单调乏味，增加一丝造型看点。衣身后背处有趣味刺绣装饰，丰富层次感，彰显别样时尚。"}
,{"content": "你好，你是谁", "summary": "你好，我是菜菜"}
,{"content": "你是谁", "summary": "你好，我是菜菜，一个帅气的偶像。"}
,{"content": "菜菜是谁", "summary": "菜菜是一个帅气的偶像，擅长唱，跳，篮球和rap。最出名的歌曲是及你太美。"}
,{"content": "介绍下菜菜", "summary": "菜菜是一个帅气的偶像，擅长唱，跳，篮球和rap。最出名的歌曲是及你太美。"}
,{"content": "菜菜", "summary": "菜菜是一个帅气的偶像，擅长唱，跳，篮球和rap。最出名的歌曲是及你太美。"}]

### Steps To Reproduce

是根据repo的readme的操作的，修改的是训练和测试数据集

### Environment

```markdown
- OS:windows11
- Python:3.9
- Transformers:4.27.1
- PyTorch:1.12.1
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :True
```
